### PR TITLE
Cleanup file permissions

### PR DESCRIPTION
**Pull request by SgtSilvio**
Instead of 770 for all files in /opt/hivemq:
- 770 for directories
- 660 for files
- 770 for executable files (.sh, mqtt-cli, hivemq-ese-helper)
Explicitly specify permissions for entrypoint scripts
- 644 for sourced files
- 755 for run files

### CODE REVIEW

**Comment by Donnerbart** [comment]
Reminder:
* 4 read permission is granted
* 2 write permission is granted
* 1 execute permission is granted

**Comment by Donnerbart** [comment]
I've kicked off the Jenkins CI again, because we just merged new K8s tests with extensions (Platform Operator only).

**Comment by Donnerbart** [comment]
> 644 for sourced files
> 755 for run files

Do we need the "others" permissions for run and sources files?
Or would 640 and 750 work as well?

**Review by LukasBrand — APPROVED** [review]

**Comment by SgtSilvio** [comment]
> Do we need the "others" permissions for run and sources files?
Or would 640 and 750 work as well?

As we require to run with groupid 0, this would work as well.
It does not hurt though because read and execute permissions do not harm on these files.

I basically only removed unnecessary/confusing execute permissions on the files in /opt/hivemq.
The permissions for the entrypoint scripts were not touched, they are now only explicitly specified.
I wanted to keep the changes to a minimum.

**Comment by SgtSilvio** [comment]
In general all permissions should be 755 for dirs and executable files and 644 for all other files.
But we require write permissions in /opt/hivemq so this folder is treated special with read==write permissions.
The "other" permissions are here 0 to avoid hivemq starting with random uid + gid != 0, but only functioning partially.

I don't like to apply this workaround to all other files. Of course we could do it for the entrypoint shell scripts, but we could then also do it for opt/java etc.
So TLDR, let's keep the special treatment only to the /opt/hivemq folder.

**Review by Donnerbart — APPROVED** [review]
